client/src/app/layout/SidebarApp/SidebarApp.tsx

```diff
@@ -227,6 +227,14 @@ export const SidebarApp: React.FC = () => {
                   </NavItem>
                 </NavExpandable>
               ) : null}
+              <NavItem>
+                <NavLink
+                  to={Paths.questionnaires}
+                  activeClassName="pf-m-current"
+                >
+                  Assessment
```

Comment:
I suppose this is design feedback for Justin but it seems confusing to me that we'll now have two views called Assessment - one here and one as a tab in application inventory. I guess because this is under admin it makes sense, but I wonder if we should propose that the nav item and page title are called "Questionnaires" instead, and a subtitle in the page header could be used to explain that they are for assessment.

Reply:
@mturley,
My understanding is that the Administrator -> Assessment page is actually displaying the questionnaires but the Required button activate those questionnaires to the default Assessment process.
So the Questionnaires.tsx page might actually not the correct name. I wonder if I should rename it now or wait.
Because this page handles Assessment settings.

Reply:
yeah maybe Assessment Settings is a better name than Assessment then. I just find the current name potentially confusing to users.

Reply:
@ibolton336, what do you think about renaming that page Assessment.tsx ?
Because the used case is about providing Administrator a way to control assessment, which starts with activating (Required column) questionnaires.

Reply:
I still find that to be a naming conflict. I'd rather it be something like AssessmentTools/QuestionaireSettings/AssessmentSettings etc.